bokehjs/src/coffee/callback/customjs.coffee

```diff
@@ -1,5 +1,5 @@
 _ = require "underscore"
-Model = require "../models/model"
+Model = require "../model/model"
```

Comment:
I hope this is temporary change and we will consistently use plural form across bokeh/bokehjs.